extensions/ql-vscode/src/extension.ts

```diff
@@ -1185,6 +1188,10 @@ function addUnhandledRejectionListener() {
       extension && getErrorStack(error).includes(extension.extensionPath);
 
     if (isFromThisExtension) {
+      if (error instanceof UserCancellationException && error.silent) {
+        return;
```

Comment:
Instead of ignoring it completely we could log it, like we do in `commands.ts`:
https://github.com/github/vscode-codeql/blob/9777cb1c6666a2d83e58a376bebe5a17834e175a/extensions/ql-vscode/src/common/vscode/commands.ts#L62-L68